Controllers/Review.cs

```diff
@@ -0,0 +1,26 @@
+﻿
+namespace University_advisor.Controllers
+{
+    public class Review
+    {
+        // laukai public, kad galėtų serialize
+        public Subject subject { get; }
+        public string author { get; }
+        public string comment { get; }
+        public int rating { get; }
+        public int id { get; }
+        private static int idNr = 0;
+        // čia kol kas labai negerai padaryti id, nes kiekvieną kartą iš naujo paleidus programą nusiresetina
```

Comment:
kadangi turim skaityma is failo, tai galime paimti paskutini irasa ir inkrementuoti ta ID

Reply:
pritariu, tą ir bandžiau padaryt, bet kol kas nebaigiau